mxcube3/ui/actions/queue.js

```diff
@@ -238,73 +257,80 @@ export function sendStopQueue() {
 }
 
 
-export function sendMountSample(queueID) {
-  return function (dispatch) {
-    fetch(`mxcube/api/v0.1/sample_changer/${queueID}/mount`, {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json'
-      }
-    }).then((response) => {
-      if (response.status >= 400) {
-        throw new Error('Server refused to mount sample');
-      } else {
-        dispatch(mountSample(queueID));
-      }
-    });
-  };
+export function setQueueAction(queue) {
+  return { type: 'SET_QUEUE', queue };
```

Comment:
Is this ever called? Should  the client not get the state from server on start? I guess this would be one of the big benefits of this is to exchange queue between server and client?